pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java

```diff
@@ -385,6 +388,15 @@ private List<KeyValuePairDto<String, String>> getLangaueNamePairs(JsonNode respo
 		List<KeyValuePairDto<String, String>> langaueNamePairs = new ArrayList<KeyValuePairDto<String, String>>();
 		List<KeyValuePairDto<String, String>> langaueNamePairsfullName = new ArrayList<KeyValuePairDto<String, String>>();
 		KeyValuePairDto<String, String> langaueNamePair = null;
+		String update = "UPDATE";
+		String userService = responseNode.get("userService").toString();
+		userService = userService.substring(1, userService.length() - 1);
+		if(userService.equalsIgnoreCase(update)){
+			fullName="surnameCop";
```

Comment:
Please add new configuration for this

Reply:
if(userService.equalsIgnoreCase(update)){
			fullName="surnameCop";
		}
		else {
			fullName=defaultFullName;
		}
		for (String name : fullName.split(",")) {

Reply:
added if(userService.equalsIgnoreCase(update)){
			fullName=defaultFullNameCop;
		}
		else {
			fullName=defaultFullName;
		}